cmd/version.go

```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+  "fmt"
+
+  "github.com/spf13/cobra"
+)
+
+func init() {
+  rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+  Use:   "version",
+  Short: "Print the version number of TK8",
+  Long:  `All software has versions. This is TK8's`,
+  Run: func(cmd *cobra.Command, args []string) {
+    fmt.Println("0.0.1")
```

Comment:
instead if printing version, add,

```
var VERSION = "v0.0.1"
```

it will be convenient to update the version

Reply:
Sure

Reply:
FYI the current release version is 0.0.3